### PR TITLE
Feature/new logger

### PR DESCRIPTION
**Pull request by gocarlos**
added miniglog and removed glog dependency

### CODE REVIEW

**Comment by gocarlos** [comment]
@nidegen  could you check this on iOS before merging?

**Comment by nidegen** [comment]
i added minor fixes, i hope the miniglog still builds for other system. i removed a casting in order to build for ios

**Review by nidegen — APPROVED** [review]
lgtm. thanks!

File: CMakeLists.txt [hunk 1]
```diff
@@ -49,26 +49,29 @@ INCLUDE_DIRECTORIES(PUBLIC "${CMAKE_CURRENT_LIST_DIR}/include")
...
 # FIND_LIBRARY(OpenGL_LIBRARIES OpenGL)
 # SET(OpenCV_DEBUG_LIBRARIES ${OpenCV_LIBRARIES})
-INCLUDE_DIRECTORIES("/usr/local/include")
-INCLUDE_DIRECTORIES("${CMAKE_CURRENT_LIST_DIR}/thirdparty/eigen")
-
-INCLUDE("${CMAKE_CURRENT_LIST_DIR}/thirdparty/slam_engine/ORB_SLAM2/CMakeLists.txt")
 
 INCLUDE_DIRECTORIES(
+  ${CMAKE_CURRENT_LIST_DIR}/thirdparty/eigen
+  ${EIGEN3_INCLUDE_DIR} # TODO: is this still needed?
```
Comment (nidegen) [inline CMakeLists.txt]: actually this is not needed i think

**Pull request merged**